### ISSUE TITLE
homebrew: xml-reader.c: <libxml/xmlreader.h> not found

### ISSUE THREAD

**Issue by kirushyk** [chergert/gnome-builder#56]
Trying to build builder against OS X using brew:
    xml-reader.c:21:10: fatal error: 'libxml/xmlreader.h' file not found

**Comment by chergert** [comment]
libxml/xmlreader.h was added in libxml2 2.5.0, and we require 2.9.0 in configure.ac:199

**Comment by kirushyk** [comment]
Sure, and I have it in my brew installation:
    $ find /usr/local/ -name xmlreader.h
    /usr/local//Cellar/libxml2/2.9.3/include/libxml2/libxml/xmlreader.h

**Comment by kirushyk** [comment]
However,

    $ pkg-config --cflags libxml-2.0
    -I/usr/include/libxml2

**Comment by chergert** [comment]
What happens if you do this:

```sh
cd contrib/xml
make clean
make V=1
```

I don't see anything weird with our $XML_CFLAGS usage ...

**Comment by chergert** [comment]
And just to double check, /usr/include/libxml2/libxml/xmlreader.h does exist right?

**Comment by kirushyk** [comment]
No, as I said, xmlreader.h is in different path than one provided by `pkg-config`.

**Comment by kirushyk** [comment]
I managed to compile this by forcelinking libxml2:

    brew link --force libxml2

**Comment by chergert** [comment]
Out of curiosity, do you know what version of libxml2 is system provided?

**Comment by kirushyk** [comment]
Default libxml2 for OS X 10.11.3 provided by Apple has version 2.9.2 but named `libxml2.2.tbd`.
This default version has `libxml/xmlreader.h` header.
